src/ast-utils/docs.ts

```diff
@@ -69,7 +69,7 @@ function annotate(descriptor) {
 /**
  * Annotates @event, @hero, & @demo tags
  */
-function annotateElementHeader(descriptor) {
+export function annotateElementHeader(descriptor: ElementDescriptor) {
```

Comment:
return type? In this case there's one path that returns and the rest don't... weird.